keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -104,7 +104,14 @@ kpxcForm.init = function(form, credentialFields) {
     if (!form.getAttribute('kpxcForm-initialized') && (credentialFields.password || credentialFields.username)) {
         form.setAttribute('kpxcForm-initialized', true);
         kpxcForm.setInputFields(form, credentialFields);
+        form.removeEventListener('submit', kpxcForm.onSubmit);
         form.addEventListener('submit', kpxcForm.onSubmit);
+
+        const submitButton = kpxc.getSubmitButton(form);
+        if (submitButton !== undefined) {
+            submitButton.removeEventListener('click', kpxcForm.onSubmit);
+            submitButton.addEventListener('click', kpxcForm.onSubmit);
```

Comment:
Not sure we need these remove/add pairs. See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Multiple_identical_event_listeners

Reply:
Thanks for the info. Then those pairs are really not needed.

Reply:
Yah just the "add" part is needed. It'll replace whatever is already attached with the same function signature.